src/ui/src/app/chart-details/chart-details.component.scss

```diff
@@ -3,31 +3,95 @@
 
 .chart-details {
   width: 100%;
-  margin: 3em 0;
-  padding: 0 2em;
   display: flex;
-  flex-direction: column-reverse;
-  flex-wrap: wrap;
+  flex-direction: column;
 
-  @include mappy-bp(medium) {
-    flex-direction: row;
-    flex-wrap: nowrap;
-
-    &__info {
-      padding-left: 1em;
+  &__header {
+    width: 100%;
+    margin-top: -80px;
+    margin-bottom: 40px;
+    &__background {
```

Comment:
For readability of the CSS code, can you add an space before every selector? It's helpful to follow the hierarchy in the files :)